checkstyle/suppressions.xml

```diff
@@ -171,6 +171,7 @@
     <suppress checks="" files="com.hazelcast.client.proxy.ClientQueueProxy"/>
     <suppress checks="" files="com.hazelcast.client.proxy.ClientMultiMapProxy"/>
     <suppress checks="" files="com.hazelcast.client.proxy.ClientMapProxy"/>
+    <suppress checks="" files="com.hazelcast.client.proxy.ClientExecutorServiceProxy"/>
```

Comment:
I exceeded number if imports by adding one more class to the proxy. I can suppress only related rule.